src/main.cc

```diff
@@ -513,6 +518,7 @@ tarantool_free(void)
 	memory_free();
 	random_free();
 #endif
+	systemd_shutdown();
```

Comment:
I've guessed that because antonym for `init` is `shutdown`. `free` is antonym for `allocate`